oxidation/libparsec_python/src/lib.rs

```diff
@@ -1,5 +1,7 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
 
```

Comment:
libparsec_python will never be compiled for wasm32 so no need for this

Reply:
I've added a comment